src/tools/w3dviewer/w3dview.cpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief W3D View
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "w3dview.h"
+#include "assetmgr.h"
+#include "gamemath.h"
+#include "mainfrm.h"
+#include "resource.h"
+#include "w3dviewdoc.h"
+#include "w3dviewview.h"
+
+CW3DViewApp theApp;
+
+BOOL CW3DViewApp::InitInstance()
+{
+    // do later
+    // ColorPickerClass::RegisterWndClass(AfxGetInstanceHandle());
+    // ColorBarClass::RegisterWndClass(AfxGetInstanceHandle());
+    GameMath::Init();
+    CWinApp::SetRegistryKey("Westwood Studios");
```

Comment:
I don't think we should use this company name.

Reply:
This is only so that our w3dviewer is compatible when it comes to stored settings/recent files/etc with the WW one (something I want)